js/main.js

```diff
@@ -358,7 +358,7 @@ class BrowserMod {
     this._motionTimeout = setTimeout(() => {
       this._fullyMotion = false;
       this.update();
-    }, 5000);
+    }, 500);
```

Comment:
This is not how often updates are sent. It's the cooldown. When motion is detected, that's sent immediately. But if no motion is detected, the clear signal is not sent for five seconds. Most motion detectors have something like this, but usually with a much longer time.

Reply:
Fixed. Sorry for not understanding,.